src/components/UserMenu/ProfilesMenu/ProfilesMenu.tsx

```diff
@@ -0,0 +1,65 @@
+import classNames from 'classnames';
+import type { UseMutateFunction } from 'react-query';
+
+import styles from '../UserMenu.module.scss';
+
+import LoadingOverlay from '#components/LoadingOverlay/LoadingOverlay';
+import MenuButton from '#components/MenuButton/MenuButton';
+import Plus from '#src/icons/Plus';
+import ProfileCircle from '#src/icons/ProfileCircle';
+import type { Profile } from '#types/account';
+
+type ProfilesMenuProps = {
+  profiles: Profile[];
+  currentProfile?: Profile;
+  small?: boolean;
+  selectingProfile: boolean;
+  selectProfile: UseMutateFunction<unknown, unknown, { id: string; avatarUrl: string }, unknown>;
+  defaultAvatar: string;
+  createButtonLabel?: string;
+  switchProfilesLabel?: string;
+  onCreateButtonClick: () => void;
+};
+
+const ProfilesMenu = ({
+  profiles,
+  currentProfile,
+  small,
+  selectingProfile,
+  selectProfile,
+  defaultAvatar,
+  createButtonLabel,
+  switchProfilesLabel,
+  onCreateButtonClick,
+}: ProfilesMenuProps) => (
+  <>
```

Comment:
This component still creates invalid HTML markup using the LoadingOverlay and `hr` element inside an `ul`

Reply:
I see that the UserMenu also contains invalid HTML markup 🤭